graph/iid/InfixIID.java

```diff
@@ -26,7 +26,7 @@
 
 public abstract class InfixIID<EDGE_ENCODING extends Encoding.Edge> extends IID {
 
-    static final int LENGTH = 1;
+    public static final int LENGTH = 1;
```

Comment:
We will use the infix length to compute a good prefix extractor when defining bloom filters, so we make it public